druid-shell/src/windows/mod.rs

```diff
@@ -899,7 +899,7 @@ unsafe fn choose_adapter(factory: *mut IDXGIFactory2) -> *mut IDXGIAdapter {
         if !SUCCEEDED((*factory).EnumAdapters(i, &mut adapter)) {
             break;
         }
-        let mut desc: DXGI_ADAPTER_DESC = mem::uninitialized();
+        let mut desc = mem::MaybeUninit::<DXGI_ADAPTER_DESC>::zeroed().assume_init();
```

Comment:
I think the way I'd write this is:

```rust
    let desc = mem::MaybeUninit::uninit();
    let hr = (*adapter).GetDesc(desc.as_mut_ptr());
    ...check hr...
    let desc = desc.assume_init();
```

I'm noticing that the existing code is not checking the return code. This is UB if the call fails.